langchain/src/agents/chat_convo/index.ts

```diff
@@ -109,13 +119,23 @@ export class ChatConversationalAgent extends Agent {
     for (const step of steps) {
       thoughts.push(new AIChatMessage(step.action.log));
       thoughts.push(
-        new HumanChatMessage(
+        new AIChatMessage(
```

Comment:
ive gone back and forth on this, making it think it wrote the answer to the tool might occasionally help with it thinking human already knows the results of tools "remember i have forgotten all text"